GNUmakefile

```diff
@@ -356,7 +356,7 @@ test-envoy-integ: $(ENVOY_INTEG_DEPS)
 test-connect-ca-providers:
 ifeq ("$(CIRCLECI)","true")
 # Run in CI
-	gotestsum --format=short-verbose --junitfile "$(TEST_RESULTS_DIR)/gotestsum-report.xml" -- ./agent/connect/ca
+	gotestsum --format=short-verbose --junitfile "$(TEST_RESULTS_DIR)/gotestsum-report.xml" -cover -coverprofile=coverage.txt -- ./agent/connect/ca
```

Comment:
These flags will need to go after the `--` since they are flags to `go test`, not to `gotestsum`.

Reply:
```suggestion
	gotestsum --format=short-verbose --junitfile "$(TEST_RESULTS_DIR)/gotestsum-report.xml" -- -cover -coverprofile=coverage.txt ./agent/connect/ca
```